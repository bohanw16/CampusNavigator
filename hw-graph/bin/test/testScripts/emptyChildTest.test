# emptyChild testing file
# Tests that a graph can be created,
# a node can be added to the graph,
# and an empty children list of the node can be printed out

# Create a graph
CreateGraph g1

# Add a node
AddNode g1 n1

# Print the nodes in the graph and the children of n1
ListNodes g1
ListChildren g1 n1