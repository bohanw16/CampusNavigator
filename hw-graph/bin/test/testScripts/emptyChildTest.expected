# emptyChild testing file
# Tests that a graph can be created,
# a node can be added to the graph,
# and an empty children list of the node can be printed out

# Create a graph
created graph g1

# Add a node
added node n1 to g1

# Print the nodes in the graph and the children of n1
g1 contains: n1
the children of n1 in g1 are: